safe/account.go

```diff
@@ -72,6 +72,7 @@ func (acc *Account) syncHistory(otherHistory accountHistory) {
 	}
 	// Sort our history to preserve order
 	historySorted := make(accountHistory, 0, len(acc.History))
+	// nolint: megacheck
```

Comment:
What's this directive here for?

Reply:
https://github.com/bndw/pick/pull/151/files#r136432196

Reply:
To be a bit more clear, `gometalinter` suggests to replace the for loop.
`safe/account.go:75:2:warning: should replace loop with historySorted = append(historySorted, acc.History...) (S1011) (megacheck)`
I still prefer the loop.

Reply:
Just so we're on the same page--

```go
// Sort our history to preserve order
historySorted := make(accountHistory, 0, len(acc.History))
// nolint: megacheck
for _, account := range acc.History {
    historySorted = append(historySorted, account)
}
sort.Sort(historySorted)
acc.History = historySorted
```

could become
```go
// Sort our history to preserve order
history := make(accountHistory, 0, len(acc.History))
history = append(history, acc.History...)
sort.Sort(history)
acc.History = history
```

IMHO I prefer the latter. Maybe I've gotten too used to Go.